Decide proposal only when excerpt is returned

njohner opened the pull request:
When an agendaitem is decided, the proposal should not be set to decided yet, as the decision has not been communicated yet (excerpt returned). We now only set the proposal to decided when the excerpt has been generated and returned.

Closing a meeting now requires that not only all agendaitems are decided, but also that the excerpts have been generated and returned for the agentaitems with proposals.

resolves #4331

deiferni commented on opengever/meeting/tests/test_proposalhistory.py:
```diff
@@ -286,11 +286,19 @@ def test_reopening_creates_history_entry(self, browser):
         self.login(self.meeting_user, browser)
         scheduling_url = self.meeting.model.get_url(view='unscheduled_proposals/1/schedule')
         deciding_url = self.meeting.model.get_url(view='agenda_items/2/decide')
+        generate_excerpt_url = self.meeting.model.get_url(view='agenda_items/2/generate_excerpt?excerpt_title=bla')
```
i get a feeling there should be a `get_url` method on an agenda item 🤔

njohner replied:
Yes I guess we could. It's used only in the tests though

deiferni commented on opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po:
```diff
@@ -806,7 +806,7 @@ msgstr "Sitzung stornieren"
 #: ./opengever/meeting/model/meeting.py
 msgid "label_close_error_has_undecided_agenda_items"
-msgstr "Die Sitzung kann nicht abgeschlossen werden solange noch nicht alle Traktanden abgeschlossen sind."
+msgstr "Die Sitzung kann nicht abgeschlossen werden solange noch nicht alle Traktanden abgeschlossen sind, die Protokollauszüge generiert und zurückgesendet."
```
this sounds a bit weird 😉. i would suggest the following changes: "Die Sitzung kann nicht abgeschlossen werden solange noch nicht alle Traktanden abgeschlossen **sind** und die Protokollauszüge generiert und zurückgesendet **wurden**."

njohner replied:
You know, my German...

njohner commented:
So I fixed the close meeting button, so that it can only be clicked when all agendaitems have been decided and excerpts returned. I also fixed up the help message below the button:

Button not available if excerpts have not been sent back:
![screen shot 2018-09-06 at 11 40 37](https://user-images.githubusercontent.com/7374243/45149862-d5022d80-b1ca-11e8-98a7-de97ecc10914.png)

Closing the meeting in that case raises an error:
![screen shot 2018-09-06 at 11 40 55](https://user-images.githubusercontent.com/7374243/45149863-d59ac400-b1ca-11e8-963d-39ac553e23ad.png)

Once excerpts have been sent back, the button is available:
![screen shot 2018-09-06 at 11 41 04](https://user-images.githubusercontent.com/7374243/45149864-d59ac400-b1ca-11e8-8789-4f671037a950.png)
